crates/biome_cli/src/commands/mod.rs

```diff
@@ -110,35 +110,35 @@ pub enum BiomeCommand {
     /// Runs formatter, linter and import sorting to the requested files.
     #[bpaf(command)]
     Check {
-        /// Writes safe fixes, formatting and import sorting
+        /// Apply safe fixes, formatting and import sorting
         #[bpaf(long("write"), switch)]
         write: bool,
 
-        /// Allow to do unsafe fixes, should be used with `--write` or `--fix`
+        /// Apply unsafe fixes. Should be used with `--write` or `--fix`
         #[bpaf(long("unsafe"), switch)]
         unsafe_: bool,
 
         /// Alias for `--write`, writes safe fixes, formatting and import sorting
         #[bpaf(long("fix"), switch, hide_usage)]
         fix: bool,
 
-        /// Allow to enable or disable the formatter check.
+        /// Allow enabling or disabling the formatter check.
         #[bpaf(
             long("formatter-enabled"),
             argument("true|false"),
             optional,
             hide_usage
         )]
         formatter_enabled: Option<FormatterEnabled>,
-        /// Allow to enable or disable the linter check.
+        /// Allow enabling or disable the linter check.
```

Comment:
There are more occurrences, including here:

```suggestion
        /// Allow enabling or disabling the linter check.
```